### PR TITLE
Patch window.addEventListener and removeEventListener

### PR DESCRIPTION
**Pull request by Artur-**
Fixes https://github.com/webcomponents/webcomponentsjs/issues/763

### CODE REVIEW

**Comment by Artur-** [comment]
Right, I included the compiled files and some more tests. Now the build seems to pass

File: tests/event-path.html [hunk 1]
```diff
@@ -236,6 +264,33 @@
...
       el.parentNode.removeChild(el);
     });
+    test('window events retarget', function() {
+      var el = createEnabledElement('x-event-scoped-window');
+      el.fireComposed();
+      assert.equal(el.windowEvents[0].target, el);
+      el.parentNode.removeChild(el);
+    });
+
+    test('window remove event listener', function() {
```
Comment (valdrinkoshi) [inline tests/event-path.html]: 💯 🍻

File: tests/event-path.html [hunk 2]
```diff
@@ -236,6 +264,33 @@
...
+      window.removeEventListener("composed", listener);
+      el.fireComposed();
+      assert.equal(1, el.windowListenerEvents);
+      el.parentNode.removeChild(el);
+    });
+
+    test('scoped event does not reach window', function() {
+      var el = createEnabledElement('x-event-scoped-window');
+      el.fireScoped();
+      assert.equal(0, el.windowEvents.length);
```
Comment (valdrinkoshi) [inline tests/event-path.html]: 🔕 👌

**Comment by sorvell** [comment]
LGTM, thx!

**Pull request merged**